.i18n-check.yaml

```diff
@@ -0,0 +1,17 @@
+# Configuration file for i18n-check validation.
+# See https://github.com/activist-org/i18n-check for details.
+
+src-dir: frontend
+i18n-dir: frontend/i18n
+i18n-src: frontend/i18n/en-US.json
+
+file-types-to-check: [.js, .ts, .vue]
+
+checks:
+  # Global configurations are applied to all checks.
+  global:
+    active: True
+    directories-to-skip: [.nuxt, .output, dist, node_modules]
+    files-to-skip: []
+  key-identifiers:
+    directories-to-skip: [test, test-e2e, test-utils]
```

Comment:
This is what was needed to make sure that test directories weren't accounted for in key identifiers, @OmarAI2003 :)